tox.ini

```diff
@@ -19,3 +19,12 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:tests]
+deps =
+    -r {toxinidir}/tests/requirements.txt
+passenv = 
+    SSH_CONFIG_FILE
+    TEST_HOSTS_LIST
+commands =
+    pytest --ssh-config={env:SSH_CONFIG_FILE:{toxinidir}/ssh.config} --hosts="{env:TEST_HOSTS_LIST:bootstrap}" {posargs} tests
```

Comment:
- What happens when you don't specify `hosts`?
- Will we, over time, be able to not necessarily run all tests on all hosts? As in, we'll have some tests validating things on specific hosts, locally, but many tests (including the one you add in this PR!) should be executed only once, on a host from which we can access the K8s cluster.
- The default `ssh-config` file path is a bit 'magic', undocumented. Should there be a default? Can we, alternatively, use the `<(vagrant ssh-config)` shell syntax (or something like that, forgot the right one), if needed by wrapping the `pytest` invocation in a script?

Reply:
* it wont work as it would probably default to localhost
* we can filter tests that would run on each pytest commands `pytest --hosts=worker1,worker2 -k "worker` or we can define the list of hosts inside the step file using `testinfra_hosts = ["woker1", "worker2"]`
* Yes that work if we are in the same folder as the .vagrant else it fails, vagrant does not like running outside its env folder so I did document how to generate the ssh.config file in the README

Reply:
- OK
- OK
- `pytest --ssh-config={env:SSH_CONFIG_FILE:<(VAGRANT_CWD={toxinidir} vagrant ssh-config)} ...` ? This is kinda assuming `tox` uses `bash` to eventually run the command, not sure that's the case... If not, a simple wrapper script to invoke `pytest` could do.

Reply:
we can use bash -c to make sure we using the right shell